server/src/test/java/io/crate/planner/optimizer/rule/EliminateCrossJoinTest.java

```diff
@@ -375,4 +377,21 @@ public void test_do_not_reorder_without_a_crossjoin() throws Exception {
         assertThat(result).isNull();
     }
 
+    @Test
+    public void test_build_graph_fron_ouble_aliased_relation() {
+        var x =  """
+                SELECt * FROM
```

Comment:
very minor, sorry :)
```suggestion
                SELECT * FROM
```